hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/LazyTransferManager.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.hadoop.fs.s3a;
```

Comment:
Ditto here.. new files need the apache license text at top

Reply:
Fixed